sql/indicator_functions.sql

```diff
@@ -437,3 +437,34 @@ WHERE
     RETURN (result);
 END$$ 
 DELIMITER ;
+
+DROP FUNCTION IF EXISTS PECG_Indicator20;
+
+DELIMITER $$
+CREATE FUNCTION PECG_Indicator20(
+    p_startDate DATE,
+    p_endDate DATE,
+    p_startAge INT(11),
+    p_endAge INT (11),
+    p_includeEndAge TINYINT(1),
+    p_gender VARCHAR(1)) RETURNS INT(11)
+    DETERMINISTIC
+BEGIN
+    DECLARE result INT(11) DEFAULT 0;
+
+SELECT
+    COUNT(DISTINCT pat.patient_id) INTO result
+FROM
+    patient pat
+WHERE
+    patientGenderIs(pat.patient_id, p_gender) AND
+    patientAgeIsBetween(pat.patient_id, p_startAge, p_endAge, p_includeEndAge) AND
+    patientHasNotBeenEnrolledIntoHivProgramDuringReportingPeriod(pat.patient_id, p_startDate, p_endDate) AND
```

Comment:
@prashantsable1991 we need to check here that the patient has never been enrolled in the HIV program (irrespective of when the enrollment happened); we'll need a function like `patientHasNotBeenEnrolledIntoHivProgram(pat.patient_id)`